tests/integration/targets/aws_msk_config/aliases

```diff
@@ -0,0 +1,4 @@
+# reason: missing-policy
+unsupported
```

Comment:
Assuming it'll be possible to run this in CI we should wait for the updated permissions before merging.

Reply:
There is an another (not only permissions) "issue" with tests execution time, for `aws_msk_config` they are pretty fast:
```
time ansible-test integration aws_msk_config --allow-unsupported --docker
...
PLAY RECAP *********************************************************************
testhost                   : ok=22   changed=6    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

AWS ACTIONS: ['kafka:CreateConfiguration', 'kafka:DeleteConfiguration', 'kafka:DescribeConfigurationRevision', 'kafka:ListConfigurations', 'kafka:UpdateConfiguration']
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.

real	0m54.458s
user	0m2.889s
sys	0m1.704s
```
but for `aws_msk_cluster` consumes a lot of time:
```
time ansible-test integration aws_msk_cluster --allow-unsupported --docker
...
PLAY RECAP *********************************************************************
testhost                   : ok=34   changed=12   unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

AWS ACTIONS: ['ec2:CreateSubnet', 'ec2:CreateTags', 'ec2:CreateVpc', 'ec2:DeleteSubnet', 'ec2:DeleteVpc', 'ec2:DescribeAvailabilityZones', 'ec2:DescribeSubnets', 'ec2:DescribeTags', 'ec2:DescribeVpcAttribute', 'ec2:DescribeVpcClassicLink', 'ec2:DescribeVpcs', 'ec2:ModifyVpcAttribute', 'kafka:CreateCluster', 'kafka:CreateConfiguration', 'kafka:DeleteCluster', 'kafka:DeleteConfiguration', 'kafka:DescribeCluster', 'kafka:DescribeConfigurationRevision', 'kafka:GetBootstrapBrokers', 'kafka:ListClusters', 'kafka:ListConfigurations', 'kafka:ListTagsForResource', 'kafka:TagResource', 'kafka:UntagResource', 'kafka:UpdateBrokerStorage', 'kafka:UpdateMonitoring']
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.

real	45m51.672s
user	0m3.401s
sys	0m1.888s
```